Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2428,6 +2435,58 @@ def to_context(self):
             }
             return ret_value
 
+    class CustomIndicator(Indicator):
+
+        def __init__(self, indicator_name, value, dbot_score, params, prefix_str):
+            """
+            :type indicator_name: ``Str``
+            :param indicator_name: name of the indicator
+            :type value: ``Any``
+            :param value: Value of the indicator
+            :type dbot_score: ``DBotScore``
+            :param dbot_score: If custom indicator has a score then create and set a DBotScore object.
+            :type params: ``Dict(Str,Any)``
+            :param params: A dictionary containing all the param names and their values
+            :type prefix_str: ``Str``
+            :param prefix_str: Will be used as the context path prefix
+            :return: None
+            :rtype: ``None``
+            """
+
+            if not prefix_str:
+                raise ValueError('prefix_str is mandatory for creating the indicator')
+
+            self.CONTEXT_PATH = f"{prefix_str}(val.value && val.value == obj.value)"
+
+            self.value = value
+            if not isinstance(dbot_score, Common.DBotScore):
+                raise ValueError('dbot_score must be of type DBotScore')
+
+            self.dbot_score = dbot_score
+
+            DBotScoreType.set_custom_name(indicator_name)
+            INDICATOR_TYPE_TO_CONTEXT_KEY[indicator_name] = indicator_name
```

Comment:
```suggestion
            INDICATOR_TYPE_TO_CONTEXT_KEY[indicator_name.lower()] = indicator_name.capitalize()
```

Not sure that this is the best solution, but it can help different sources to use the same context keys.
Think about it, or find a better solution.

Reply:
What do you mean by different sources?